src/git_url_parser.rs

```diff
@@ -18,7 +18,8 @@ pub fn parse_url(url: &str) -> Result<GitUrl, GitDownError> {
 }
 
 pub fn is_shortcut_url(url: &str) -> bool {
-    let regex = Regex::new(r"^\w+:\w+*$").unwrap();
+    // FIXME: Do not rely on hardcoded forms of the shortcuts. 
+    let regex = Regex::new(r"^(bb|gh|gl|sf):.*$").unwrap();
```

Comment:
This works... After all, those are the only acceptable shortcuts at the moment. Missed a '.' in the original regex. Should have been `^\w+:\w+.*$`.

Been thinking of adding unit tests. Will add some when I find time.

Reply:
Yeah, was also thinking of adding tests but my Rust is.... rusty 🥁